src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+// Беремо всі хедери у масив, щоб додати до кожного ивент
+const headers = [...document.querySelectorAll('th')];
+
+for (let i = 0; i < 4; i++) {
+  headers[i].addEventListener('click', () => {
+    const tbody = document.querySelector('tbody');
+    const rows = [...tbody.querySelectorAll('tr')];
+
+    rows.sort((rowA, rowB) => {
+      const rowNumberA = parseFloat(rowA.cells[i].innerHTML.replace('$', ''));
+      const rowNumberB = parseFloat(rowB.cells[i].innerHTML.replace('$', ''));
+
+      if (rowNumberA && rowNumberB) {
```

Comment:
The condition `if (rowNumberA && rowNumberB)` may not work as intended if the cell values are zero or other falsy values. Consider using `if (!isNaN(rowNumberA) && !isNaN(rowNumberB))` to ensure that both values are valid numbers.